build_tools/benchmarks/common/benchmark_suite_test.py

```diff
@@ -127,8 +124,7 @@ def test_load_from_benchmark_suite_dir(self):
               category="PyTorch",
               available_drivers=["vulkan"],
               available_loaders=[],
-              cpu_target_arch_filter="cpu-armv8",
-              gpu_target_arch_filter="gpu-mali"), [case2])
+              target_architectures=["cpu-armv8", "gpu-mali"]), [case2])
 
   def test_load_from_run_configs(self):
     model_tflite = common_definitions.Model(
```

Comment:
This is failing on macOS: https://github.com/openxla/iree/actions/runs/4868571511/jobs/8682175725#step:7:2529

Reply:
I'm looking into it.

Reply:
Fix: #13390